pets/common/views.py

```diff
@@ -7,7 +7,9 @@ class AboutPageView(TemplateView):
 
 
 def not_found(request):
-    return render(request, 'staticpages/404.html')
+    response = render(request, 'staticpages/404.html')
+    response.status_code = 404
+    return response
```

Comment:
You could have `404.html` at the top-level template tree and use [`Http404`](https://docs.djangoproject.com/en/1.9/topics/http/views/#the-http404-exception) instead of this method.